bftengine/src/bftengine/PersistentStorage.hpp

```diff
@@ -109,6 +110,8 @@ class PersistentStorage {
   virtual void setCheckpointMsgInCheckWindow(SeqNum seqNum, CheckpointMsg *msg) = 0;
   virtual void setCompletedMarkInCheckWindow(SeqNum seqNum, bool mark) = 0;
 
+  virtual void setUserData(const void *data, std::size_t numberOfBytes) = 0;
```

Comment:
Please add an explanation what is the purpose of this data.

Reply:
I've added a general description of what that data might be, e.g. application-specific local data, scratchpad data, etc.